topics/admin/tutorials/heterogeneous-compute/tutorial.md

```diff
@@ -254,34 +255,9 @@ We also need to create the dependency resolver file so pulsar knows how to find
 >
 > 2. Log in to the machines and have a look in the `/mnt/pulsar` directory. You will see the venv and config directories. All the config files created can be perused.
 >
-{: .hands_on}
-
-Pulsar can now be started. We do that by first activating the virtualenv, then running it and telling it which webserver to use (Paster) and where the config files are.
-
-> ### {% icon hands_on %} Hands-on: Starting Pulsar
->
-> 1. Login to your Pulsar server
->
-> 2. Change directory into `/mnt/pulsar`
->
-> 3. Activate the virtualenv (`source venv/bin/activate`)
->
-> 4. Run pulsar (`pulsar -m paster -c config/`)
->
-{: .hands_on}
-
-
-A log will now start scrolling, showing the startup of pulsar. You'll notice that it will be initializing and installing conda. Once it is complete and listening on the assigned port, you can stop the server with Ctrl-C.
-
-
-> ### {% icon hands_on %} Hands-on: Daemonizing
->
-> 1. This is a pretty gross way of running Pulsar and it can run in daemon mode. To do that, add `--daemon` to the end of the above command line.
->
-> 2. Stopping the daemon is as simple as re-running the command with `--stop-daemon` instead.
->
-> 3. Before proceeding, make sure your Pulsar is running in `--daemon` mode.
+> 3. Run `journalctl -f -u pulsar`
 >
+>    A log will now start scrolling, showing the startup of pulsar. You'll notice that it will be initializing and installing conda. Once it is complete and listening on the assigned port.
```

Comment:
```suggestion
>    A log will now start scrolling, showing the startup of pulsar. You'll notice that it will be initializing and installing conda. Once this is completed, Pulsar will be listening on the assigned port.
```